JitsiConference.js

```diff
@@ -2072,12 +2076,12 @@ JitsiConference.prototype._removeRemoteTracks
  * @private
  */
 JitsiConference.prototype._setP2PStatus = function(newStatus) {
-    if (this.p2pEstablished === newStatus) {
+    if (this.p2p === newStatus) {
```

Comment:
no, it should be only this method that accesses it directly and the getter for this state. But for me it's fine to access it here directly (it's a setter for this field).